UnitTests/expected/Storyboards-osx-Message-Defaults.swift.out

```diff
@@ -46,22 +46,22 @@ struct StoryboardScene {
 
     case MessageDetailsScene = "MessageDetails"
     static func instantiateMessageDetails() -> NSViewController {
-      return StoryboardScene.Message_Osx.MessageDetailsScene.controller()
+      return StoryboardScene.Message_Osx.MessageDetailsScene.controller() as! NSViewController
```

Comment:
Indeed, after rebasing I have swiftlint running when testing, still no force_cast error. Also, do you also have in master all these warnings:

<img width="356" alt="screen shot 2016-06-05 at 23 40 22" src="https://cloud.githubusercontent.com/assets/1402212/15806519/26a556f8-2b77-11e6-82f8-502e08596c34.png">